app/build.gradle

```diff
@@ -16,17 +16,19 @@ android {
     signingConfigs {
         debug {
             // keytool -genkeypair -v -keystore my-debug-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000
+            // ./gradlew assemble -Dpassword=123456
             storeFile file('my-debug-key.keystore')
-            storePassword '123456'
+            storePassword System.getProperty('password')
             keyAlias 'my-key-alias'
-            keyPassword '123456'
+            keyPassword System.getProperty('password')
         }
         release {
             // keytool -genkeypair -v -keystore my-release-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000
+            // ./gradlew assemble -Dpassword=123456
             storeFile file('my-release-key.keystore')
-            storePassword '123456'
+            storePassword System.getProperty('password')
             keyAlias 'my-key-alias'
-            keyPassword '123456'
+            keyPassword System.getProperty('password')
```

Comment:
@xinminsu Can you also add documentation explaining how to pass the `password` property through the command line?

Reply:
// ./gradlew assemble -Dpassword=123456

Reply:
@xinminsu Thanks. And can you add this documentation to the code itself, please?

Reply:
in coments of signingConfigs in app/src/build.gradle.

Reply:
in second line of signingConfigs coments  in app/src/build.gradle.